helm/freenas-provisioner/templates/roles.yaml

```diff
@@ -0,0 +1,41 @@
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: {{ include "freenas-provisioner.serviceAccountName" . }}
```

Comment:
Should be relative to release name to be able to instanciate multiple times this chart.